tests/playbooks/tests_network_state.yml

```diff
@@ -173,6 +173,14 @@
         msg: the file `/run/NetworkManager/no-stub-resolv.conf` does not
           contain the specified DNS configuration
 
+    - name: Purge the DNS
+      import_role:
+        name: linux-system-roles.network
+      vars:
+        network_state:
```

Comment:
Ah - this was the cause of the failures related to DNS not working after running the network_state test